src/cmd-buildextend-live

```diff
@@ -506,6 +506,13 @@ configfile $prefix/grub.cfg
 boot
 ''')
 
+            # Disable shim fallback; install grub as second-stage bootloader
+            # for fallback path
+            for fbpath in glob.glob(os.path.join(tmpimageefidir, "BOOT", "fb*.efi")):
```

Comment:
Minor/optional: whereas the shell version errors out if the glob doesn't expand, this one I think will silently ignore it, right? Should we be more strict here and ensure we had a match to be consistent?